src/client/eppo-client.ts

```diff
@@ -7,6 +7,7 @@ import {
   NonExpiringInMemoryAssignmentCache,
 } from '../assignment-cache';
 import { IAssignmentEvent, IAssignmentLogger } from '../assignment-logger';
+import ConfigurationRequestor from '../configuration-requestor';
```

Comment:
Note `FlagConfigurationRequestor` was renamed to `ConfigurationRequestor` since it requests flags and bandit parameters now